README.md

```diff
@@ -161,6 +161,11 @@ knife ec2 server create -I ami-d0f89fb9 --ssh-key your-public-key-id -f m1.mediu
 # A Windows instance via the WinRM protocol -- --ssh-key is still required due to EC2 API operations that need it to grant access to the Windows instance
 # `--spot-price` option lets you specify the spot pricing
 knife ec2 server create -I ami-173d747e -G windows -f m1.medium --user-data ~/your-user-data-file -x '.\a_local_user' -P 'yourpassword' --ssh-key your-public-key-id --spot-price price-in-USD
+
+# Pass --server-connect-attribute if the machine is in VPC mode
```

Comment:
I don't think this is limited to VPC, although that's where we're going to pretty much always use it.

```Pass --server-connect-attribute to specify the instance attribute that we will try to connect to via ssh/winrm```